README.md

```diff
@@ -1,73 +1,72 @@
-  - een _functie_ voor het verwijderen van 1 `Television`
-  - een _functie_ voor het updaten van 1 `Television`
-- De `Controller` is door middel van een _autowired_ gelinkt aan de `Service`;
-- Het project bevat, op de juiste plaats in de map-structuur, een map genaamd `Dtos`;
-- De map `Dtos` bevat een `TelevisionDto` en een `TelevisionInputDto`;
-- De _requestMappings_ in de `Controller` worden aangepast zodat deze de juiste response doorgeven via de `Service`;
-- De `Service` maakt gebruik van de gegevens die we via de `Controller` doorkrijgen van de `Dtos`;
-- Het project bevat de _validator_ dependency uit paragraaf 8.1 van de cursus Spring Boot in Edhub.
 
-### Belangrijk
-- Na deze les moeten de _requestMappings_ in de controller de juiste responses geven.
-- Na deze les is de applicatie voor het eerst functioneel.
+Het project bevat de volgende `Models`:
+- `Television`
+- `RemoteController` met de _variables_:
+  -_Long_`id`
+  -_String_`compatibleWith`
+  -_String_`batteryType`
+  -_String_`name`
+  -_String_`brand`
+  -_Double_ `price`
+  -_Integer_ `originalStock`
+- `CI-Module` met de _variables_:
+  -_Long_ `id`
+  -_String_ `name`
+  -_String_ `type`
+  -_Double_ `price`
+- `WallBracket` met de _variables_:
+  -_Long_ `id`
+  -_String_ `size`
+  -_Boolean_ `ajustable`
+  -_String_ `name`
+  -_Double_ `price`
+  Voor elk van deze modellen bevat je applicatie ook een `Repository`, `Controller`, `Dto`, `InputDto` en een `Service`.
 
-Het is belangrijk om goed te begrijpen hoe de lagen in onze multi-tier application samenwerken: 
-  - De `Controller` ontvangt een verzoek op een _endpoint_, als er _variable_ worden meegeven aan dit verzoek komen die binnen via de `TelevisionInputDto` ->
-  - De `Controller` geeft aan de hand van het verzoek en eventuele meegekregen _variable_ dit door aan de `Service` ->
-  - De `Service` spreekt aan de hand van de geschreven functie de `Repository` aan met de juiste _find_ functie/query ->
-  - De `Repository` gaat aan de hand van het bijbehorende `Model` zoeken naar de juiste gegevens en stuurt deze terug naar de `Service` ->
-  - De `Service` past de logica toe uit de functie en geeft de response aan de hand van de `TelevisionDto` terug aan de `Controller`
- (De controller wordt vanuit de _front-end_ of vanuit _Postman_ aangesproken, en geeft ook het antwoord weer terug.) 
+Daarnaast bevat het project:
+- Een `OneToOne` relatie tussen `Television` en `RemoteController`
+- Een `OneToMany` relatie tussen `Television` en `CI-Module`
+- Een `ManyToMany` relatie tussen `Television`en `WallBracket`
+
+### Belangrijk
+Je mag de constructors uit de modellen weglaten. Deze vult Spring Boot automatisch in als ze niet gedefinieerd zijn. Ook uit de `Television` mogen de constructors verwijderd worden.
 
 ## Stappenplan
 _Let op_: het is uitdagender om jouw eigen stappenplan te maken. Mocht je niet zo goed weten waar je moet beginnen, kun je onderstaand stappenplan volgen:
```

Comment:
Het toevoegen van de @JoinTable annotatie voor de ManyToMany relatie is goed gedaan, zorg ervoor dat je de namen consistent houdt met de database.